fuse_examples/classification/MG_CMMD/dataset.py

```diff
@@ -157,16 +157,17 @@ def merge_clinical_data_with_dicom_tags(root_path: str, target: str) -> str:
     :return: the new csv file path
     """
     input_source = root_path + '/CMMD_clinicaldata_revision.csv'
-    combined_file_path = root_path + '/files_combined.csv'
+    combined_file_path = data_misc_dir + '/files_combined.csv'
     if os.path.isfile(combined_file_path):
         return combined_file_path
+    Path(data_misc_dir).mkdir(parents=True, exist_ok=True)
     clinical_data = pd.read_csv(input_source)
     scans = []
-    for patient in os.listdir(root_path + 'CMMD/'):
-        path = root_path + 'CMMD/' + patient
+    for patient in os.listdir(root_path + '/CMMD/'):
+        path = root_path + '/CMMD/' + patient
         for dicom_file in glob.glob(os.path.join(path, '**/*.dcm'), recursive=True):
-            file = os.path.join(path, dicom_file)
-            dcm = pydicom.dcmread(file)
+            file = dicom_file[len(root_path):] if dicom_file.startswith(root_path) else ''
```

Comment:
So what id file - data_dir? why do you need it?

Reply:
we talked about it